src/patternfly/components/Table/docs/table-headers-wrap.md

```diff
@@ -0,0 +1,9 @@
+### Table expandable notes
+
+**All simple table accessibility and usage requirements apply.**
+
+### Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-m-wrap` | `<th>`, `<td>` | Modifies content to wrap. |
```

Comment:
@mcoker not sure if its necessary to be on the `<td>` here because the td wraps by default. Only thead has `white-space: nowrap;` set on it. Maybe I'm missing something.

Reply:
it's because `<td>` is valid in a `<thead>`